.github/workflows/ci.yml

```diff
@@ -69,25 +69,25 @@ jobs:
           - { compiler: clang-13,  cxxstd: '03,11,14,17,20', os: ubuntu-22.04, install: 'clang-13 g++-12', gcc_toolchain: 12 }
           - { compiler: clang-14,  cxxstd: '03,11,14,17,20', os: ubuntu-22.04, install: 'clang-14 g++-12', gcc_toolchain: 12 }
           - { compiler: clang-15,  cxxstd: '03,11,14,17,20', os: ubuntu-22.04, install: 'clang-15 g++-12', gcc_toolchain: 12 }
-          - { compiler: clang-16,  cxxstd: '11,14,17,20,2b', os: ubuntu-latest, container: 'ubuntu:24.04' }
-          - { compiler: clang-17,  cxxstd: '11,14,17,20,2b', os: ubuntu-latest, container: 'ubuntu:24.04' }
-          - { compiler: clang-18,  cxxstd: '11,14,17,20,2b', os: ubuntu-latest, container: 'ubuntu:24.04' }
+          - { compiler: clang-16,  cxxstd: '11,14,17,20,2b', os: ubuntu-24.04' }
+          - { compiler: clang-17,  cxxstd: '11,14,17,20,23', os: ubuntu-24.04' }
+          - { compiler: clang-18,  cxxstd: '11,17,20,23,2c', os: ubuntu-24.04' }
 
           # libc++
-          - { compiler: clang-7,   cxxstd: '03,11,14',       os: ubuntu-20.04, stdlib: libc++, install: 'clang-7 libc++-7-dev libc++abi-7-dev' }
-          - { compiler: clang-12,  cxxstd: '03,11,14,17,20', os: ubuntu-20.04, stdlib: libc++, install: 'clang-12 libc++-12-dev libc++abi-12-dev' }
+          - { compiler: clang-7,   cxxstd: '03,11,14,17',    os: ubuntu-20.04, stdlib: 'libc++', install: 'clang-7 libc++-7-dev libc++abi-7-dev' }
           - { name: Clang w/ sanitizers, sanitize: yes,
-              compiler: clang-12,  cxxstd: '03,11,14,17,20', os: ubuntu-20.04, stdlib: libc++, install: 'clang-12 libc++-12-dev libc++abi-12-dev' }
+              compiler: clang-12,  cxxstd: '03,11,14,17,20', os: ubuntu-20.04, stdlib: 'libc++', install: 'clang-12 libc++-12-dev libc++abi-12-dev' }
 
           - { name: MacOS w/ clang and sanitizers,
-              compiler: clang,     cxxstd: '03,11,14,17,20,2b', os: macos-13, sanitize: yes }
-          - { compiler: clang,     cxxstd: '03,11,14,17,20,2b', os: macos-14 }
+              compiler: clang,     cxxstd: '03,11,14,17,20', os: macos-13, sanitize: yes }
+          - { compiler: clang,     cxxstd: '03,11,17,20,2b', os: macos-14 }
+          - { compiler: clang,     cxxstd: '03,11,17,20,2b', os: macos-15 }
```

Comment:
Not strongly opposed but I was a bit surprised that a C++ standard in the middle is missing. I'd rather have them consecutive by default

Reply:
This was a trade-off to keep 03 in the mix.  Happy to remove 03 entirely.